src/Features/Core/Portable/CodeCleanup/CodeCleanupOptions.cs

```diff
@@ -17,6 +17,15 @@ internal static class CodeCleanupOptions
             nameof(CodeCleanupOptions), nameof(NeverShowCodeCleanupInfoBarAgain), defaultValue: false,
             storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Never Show Code Cleanup Info Bar Again"));
 
+        /// <summary>
+        /// Set default value to true for 15.8 Preview to promote this new code cleanup feature
+        /// Will set it back to false for RTM so users are not silently opting into this feature
+        /// https://github.com/dotnet/roslyn/issues/28068
+        /// </summary>
```

Comment:
Not really sure this is a good use of the `<summary>` tag...maybe just a regular comment?